.buildkite/commands/run-unit-tests.sh

```diff
@@ -18,7 +18,7 @@ echo "--- 🚦 Report Tests Status"
 results_file="WooCommerce/build/test-results/merged-test-results.xml"
 # Merge JUnit results into a single file (for performance reasons with reporting)
 # See https://github.com/woocommerce/woocommerce-android/pull/12064
-./merge_junit.sh -d WooCommerce/build/test-results/testJalapenoDebugUnitTest -o $results_file
+merge_junit_reports -d WooCommerce/build/test-results/testJalapenoDebugUnitTest -o $results_file
```

Comment:
Nitpick to make [`shellcheck`](https://github.com/koalaman/shellcheck) happy:
```suggestion
merge_junit_reports -d WooCommerce/build/test-results/testJalapenoDebugUnitTest -o "$results_file"
```